doc/sphinx/source/core/quantization.rst

```diff
@@ -0,0 +1,163 @@
+trained neural network, without any significant loss in model accuracy. Compared
+with 32-bit floating point (FP32), using arithmetic with lower precision such as
+INT8 to calculate weights and activation requires less memory. However, memory
+reduction varies based on the model (ResNet 50, Inception v3, BERT, etc.).
+
+Implementing a quantized model with nGraph
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To implement a quantized model with
+nGraph, provide a partially (or fully) quantized FP32 model (e.g., where the
+convolution layer in the model is replaced with a quantized convolution) to the
+nGraph library along with quantized parameters: weights, activations, scale, and
+zero point. 
+
+.. Note:: Currently, nGraph only supports quantization for inference.
+
+nGraph Quantized Operators (Ops)
+--------------------------------
+
+nGraph uses scale and zero point (also used by ONNX) to map real values to
+quantized values.  All quantized ops (APIs) in nGraph use scale and zero point
+and they can be used just like any other nGraph op. 
+
+**Scale**: the quantization scale for the tensor 
+
+**Zero point**: the zero point of the tensor 
+
+**Round mode**: used in combination with scale and zero point to round real 
+values to quantized values
+
+.. table:: nGraph Quantized Ops
+
+
+	+----------------------+-----------------------------------------------+
+	| Operator             | Description                                   |
+	+======================+===============================================+
+	| Quantize             | Maps real values (r) to quantized values (q)  |
+	|                      | using scale (s), zero point (z),              |
+	|                      | and round mode; produces a quantized tensor   |
+	+----------------------+-----------------------------------------------+
+	| Dequantize           | Maps quantized values (q) to real values (r)  |
+	|                      | using scale (s) and zero point (z); converts  |
+	|                      | a quantized tensor to a floating point tensor |
+	+----------------------+-----------------------------------------------+
+	| FakeQuantize         | Performs element-wise linear quantization     |
+	+----------------------+-----------------------------------------------+
+	| QuantizedConvolution | Performs quantized convolution; takes the     |
+	|                      | takes the same parameters as the              |
+	|                      | non-quantized convolution operator            |
+	+----------------------+-----------------------------------------------+
+	| QuantizedDot         | Performs quantized dot; takes the same        |
+	|                      | parameters as the non-quantized dot operator  |
+	+----------------------+-----------------------------------------------+
+
+Some frameworks such as TensorFlow have fused (or layer) ops. nGraph provides
+optional operations (listed in the following table) to help users easily
```

Comment:
There is a lot of parenthesis used in this paragraph.  If we add "experimental", do we need to say `listed in the following table).  Example;  `nGraph provides optional experimental operations to help users...`